packages/blade/src/tokens/global/motion.ts

```diff
@@ -0,0 +1,138 @@
+import makeBezier from '../../utils/makeBezier';
+
+type Duration = {
+  /** `70` milliseconds */
+  quick1: 70;
+  /** `150` milliseconds */
+  quick2: 150;
+  /** `200` milliseconds */
+  quick3: 200;
+  /** `250` milliseconds */
+  moderate1: 250;
+  /** `300` milliseconds */
+  moderate2: 300;
+  /** `400` milliseconds */
+  gentle1: 400;
+  /** `600` milliseconds */
+  gentle2: 600;
+  /** `900` milliseconds */
+  gentle3: 900;
+};
+
+type Delay = {
+  /** `70` milliseconds */
+  '2xshort': 70;
+  /** `120` milliseconds */
+  xshort: 120;
+  /** `180` milliseconds */
+  short: 180;
+  /** `3000` milliseconds */
+  long: 3000;
+  /** `5000` milliseconds */
+  xlong: 5000;
+};
+
+type Easing = {
+  /** Easings for all standard animations*/
+  standard: {
+    /** `cubic-bezier(0.5, 0, 0.3, 1.5)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    attentive: 'cubic-bezier(0.5, 0, 0.3, 1.5)' | { factory: () => (value: number) => number };
+    /** `cubic-bezier(0.3, 0, 0.2, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    effective: 'cubic-bezier(0.3, 0, 0.2, 1)' | { factory: () => (value: number) => number };
```

Comment:
In here we are hard coding the string type. 
But in the actual easing values we are using that makeBezier() function which returns the narrowed template literal string. 

If you want you can extract the type and use it here, looks much cleaner 

```ts
export type MakeBezier<
  X1 extends number,
  Y1 extends number,
  X2 extends number,
  Y2 extends number
> = `cubic-bezier(${X1}, ${Y1}, ${X2}, ${Y2})`;

const makeBezier = <X1 extends number, Y1 extends number, X2 extends number, Y2 extends number>(
  x1: X1,
  y1: Y1,
  x2: X2,
  y2: Y2,
): MakeBezier<X1, Y1, X2, Y2> => {
  return `cubic-bezier(${x1}, ${y1}, ${x2}, ${y2})`;
};

```

<img width="855" alt="Screenshot 2022-04-18 at 4 34 43 PM" src="https://user-images.githubusercontent.com/35374649/163799668-dfaa2791-fb78-4234-b2cc-08a46ba425ec.png">

Reply:
I think that's okay since these are constant values. Similar to  how `duration.quick3` has a type of `200`, this will have a type of `cubic-bezier(...)`

My goal is to ensure that `entrance.effective` should _always_ be `cubic-bezier(0, 0, 0, 1)`. If someone changes the definition of what `MakeBezier` is to something like this:
```js
export type MakeBezier<
  X1 extends number,
  Y1 extends number,
  X2 extends number,
  Y2 extends number
> = `some-incorrect-usage(${X1}, ${Y1}, ${X2}, ${Y2})`;
```

This will never throw any errors since my expectation and outcome will always be of type `MakeBezier`. Since we know what our expectation is, it's better to keep it as a constant `cubic-bezier(0, 0, 0, 1)`

Reply:
Okay cool then. makes sense.